### ISSUE TITLE
Orange 3.22.0 is the last version I can open in Mac OS 10.11.06

### ISSUE THREAD

**Issue by lawrencerowland** [biolab/orange3#4446]
**Describe the bug**
Orange 3.23.1 and 3.24.1 fail to open when installed on Mac OS 10.11.06.
Installation is okay. 
When opening for the first time, The icon appears, and then disappears after a few seconds, without error report. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to archive of previous download files
2. Download dmg file
3. Open dmg, install by moving to app folder. 
4. Click to open Orange in app folder. 

**Orange version:**
Orange 3.23.1 and 3.24.1 
I have gone back to 3.22 and that is working fine. 

**Expected behavior**
I expected Orange window to open. 

**Operating system:**
Mac OS 10.11.06

**Additional context**
Clearly this is a pretty old OS, and you may wish to close this as an issue - because there is no need you should support this far back. For myself, I can still use Orange on one machine, but not on this older machine which I use as a laptop. I have used several older versions of Orange with success through the first half of 2019.

**Comment by ales-erjavec** [comment]
Duplicate of https://github.com/biolab/orange3/issues/4323

You can try running:
```
/Applications/Orange3.app/Contents/MacOS/pip install pyqt5~=5.9.0
```
from the terminal after dragging the app to 'Applications' folder to downgrade the contained Qt library. This should make the 3.24.1 version work on macOS 10.11.06.

**Comment by lawrencerowland** [comment]
Many thanks for this. I tried it, but it didnt work on my machine, although it got further. 
Issue should stay closed - there is no need to support this old machine!
I will have another go next week, and may try to install it with pip this time - to see what is happening.

**Comment by lawrencerowland** [comment]
For the record on this one. 
I downloaded Orange 3.24.1 this time via Conda.
I then followed the instruction above.
This was the error I got, in case any one has the same problem. 
(I will reinstall Orange 3.22- it still does a lot!)

~ lawrence$ pip install pyqt5~=5.9.0
Collecting pyqt5~=5.9.0
  Using cached PyQt5-5.9.2-5.9.3-cp35.cp36.cp37-abi3-macosx_10_6_intel.whl (81.7 MB)
ERROR: Could not find a version that satisfies the requirement sip<4.20,>=4.19.4 (from pyqt5~=5.9.0) (from versions: 5.0.0, 5.0.1, 5.1.0, 5.1.1)
ERROR: No matching distribution found for sip<4.20,>=4.19.4 (from pyqt5~=5.9.0)
